bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/Utils.java

```diff
@@ -74,9 +75,16 @@ public static ZonedDateTime parseDate(String date) throws DateTimeParseException
      * @return
      */
     public static String formatDate(Instant date) {
-        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
-                .withZone(ZoneId.systemDefault());
-        return formatter.format(date);
+        if (date == OUTDATED_DATE) {
+            return "(expired)";
+        }
+        try {
+            final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                    .withZone(ZoneId.systemDefault());
+            return formatter.format(date);
+        } catch (DateTimeException ex) {
+            return date.toString();
```

Comment:
I'm sorry, one last comment when thinking about this again: If you drop this method entirely and simply use `Instant.toString()` directly, you would:
- Fix the bug.
- Simplify the code.
- Have accurate timestamps in the logs (formatting with local time-zone can lead to ambiguous timestamps because of DST).

Reply:
You are right. I will change it.